benchmarks/src/garage_benchmarks/helper.py

```diff
@@ -211,3 +218,24 @@ def _export_to_json(json_name, xs, xlabel, ys, ylabel, ys_std):
                  y_max=(ys + ys_std).tolist(),
                  xlabel=xlabel,
                  ylabel=ylabel), json_file)
+
+
+def _upload_to_gcp_storage(exec_dir):
```

Comment:
i think you should be more conservative here and upload the entire experiment directory.

Reply:
Can you develop your point? The rest is not being used by the auto benchmarking.

Reply:
What if you want to give users the opportunity to download the pre-trained policy from the website?
What if we see a regression and want to dig deeper and see if it was a code problem, or an unlucky seed?

Reply:
Yeah, I agree. As long as we use `snapshot_mode='last'`, we don't have any reason not to upload the entire experiment directory. It'll definitely help with debugging, and will cost us mere cents / month.